lib/platform/azure/index.js

```diff
@@ -522,8 +533,10 @@ async function addReviewers(prNo, reviewers) {
 }
 
 // istanbul ignore next
-function deleteLabel() {
-  throw new Error('deleteLabel not implemented');
+async function deleteLabel(prNumber, label) {
+  logger.debug(`Deleting label ${label} from #${issueNo}`);
+  const azureApiGit = await azureApi.gitApi();
+  await azureApiGit.deletePullRequestLabels(config.repoId, prNumber, label);
```

Comment:
Make sure it doesn't throw an error if the label isn't found

Reply:
It catches the error internally. See [here](https://github.com/Microsoft/azure-devops-node-api/blob/master/api/GitApi.ts#L3284-L3330)